ee/vellum_cli/__init__.py

```diff
@@ -70,6 +113,35 @@ def pull(
     )
 
 
+@pull.command(name="*")
+@click.pass_context
+@click.option(
+    "--include-json",
```

Comment:
nit: It'd be cool to DRY out some of these options/help text

Reply:
I started to do so, but the wording was slightly different depending on Resource or Workflow. may still try to do so in a future pr